src/blueprint/README.md

```diff
@@ -0,0 +1,153 @@
+# Azure CLI blueprint Extension #
+This package is for the 'blueprint' extension, i.e. 'az blueprint'. More info on what is [Blueprint](https://docs.microsoft.com/azure/governance/blueprints/overview).
+
+### How to use ###
+Install this extension using the below CLI command
+```
+az extension add --name blueprint
+```
+
+Blueprint can be scoped in a subscription or management group, which is a group of subscriptions. You can use management group by specify `--management-group`, or use subscription by specify `--subscription.` If both parameters are omitted, the command will use your default subscription. You can set it by running:
+```
+az account set --subscription subscription_id
+```
+
+### Included Features
+#### Blueprint Definition Management:
+*Examples:*
+
+##### Import Blueprint Definition and Artifacts Settings
+
+```
+az blueprint import \
+--name blueprintName \
+--input-path "/path/to/blueprint/directory"
+
+```
+
+In the input directory, there should be a file named "blueprint.json" that has blueprint definition and parameters for artifacts. There should be a subdirectory named "artifacts" and it should contain files for artifact definition. Examples can be found [here]().
+
+The import command will overwrite defnitions for blueprint and artifacts if a blueprint with the same name already exists.
+
+##### Create a Blueprint with Parameters
+
+```
+az blueprint create \
+    --name blueprintName \
+    --description "An example blueprint." \
+    --target-scope "subscription" \
+    --parameters @/path/to/blueprint_params.json
```

Comment:
Just curious, does this only available from command line? Because I didn't find the place to create a blueprint with params in the Azure portal, or maybe I missed something.

Reply:
The parameters come from artifacts. In Azure portal, you create all artifacts and a blueprint together. So the portal adds the paramters from artifacts for you.